Workbooks/Azure Sentinel/Overview/SentinelOverview.workbook

```diff
@@ -0,0 +1,2091 @@
+              "size": 0,
+              "queryType": 0,
+              "resourceType": "microsoft.operationalinsights/workspaces",
+              "visualization": "linechart",
+              "chartSettings": {
+                "xAxis": "TimeGenerated",
+                "yAxis": [
+                  "incedentAmount"
+                ],
+                "createOtherGroup": 15
+              }
+            },
+            "name": "Incidents created by rules"
+          }
+        ]
+      },
+      "conditionalVisibility": {
+        "parameterName": "SelectedTab",
+        "comparison": "isEqualTo",
+        "value": "Analytics"
+      },
+      "name": "ANALYTICS GROUP"
+    }
+  ],
+  "fallbackResourceIds": [
+    ""
+  ],
+  "fromTemplateId": "sentinel-UserWorkbook",
```

Comment:
we can delete "fallbackResourceIds" and "fromTemplateId". These are auto generated when we author workbooks from existing templates, but since we want to create this as a template, we don't need these fields. I will update the documentation for this